.github/SUPPORT.md

```diff
@@ -0,0 +1,16 @@
+# Jekyll Support
+
+## Getting Help
+
+If you're looking for support for Jekyll there are a lot of options, please check out:
```

Comment:
Would it be worth being specific here, e.g., "Jekyll's issue tracker is not a support forum"?